### PR TITLE
Adding formatting to exported data

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Adds sorting by name and datetime to exported data.
- [x] Adds calibrated and raw data to the exported data.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-1119](https://airqoteam.atlassian.net/browse/PLAT-1119)

**_HOW DO I TEST OUT THIS PR?_**
Follow instructions on the microservice readme file

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

[Analytics Download endpoint from BigQuery](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/analytics)

### CODE REVIEW

File: src/analytics/api/models/events.py [hunk 1]
```diff
@@ -25,13 +25,20 @@ def __init__(self, tenant):
     @cache.memoize()
     def from_bigquery(cls, tenant, sites, start_date, end_date, frequency, pollutants):
         decimal_places = 2
-        formatted_pollutants = [f'ROUND({pollutant}, {decimal_places}) AS {pollutant}' for pollutant in pollutants]
+        pollutant_mappings = dict({
+            "pm2_5": ["pm2_5_calibrated_value", "pm2_5_raw_value"],
+            "pm10": ["pm10_calibrated_value", "pm10_raw_value"],
+            "no2": ["no2_calibrated_value", "no2_raw_value"],
+        })
```
Comment (NoahNsimbe) [inline src/analytics/api/models/events.py:32]: Adding calibrated and raw values to the export

File: src/analytics/api/models/events.py [hunk 2]
```diff
@@ -25,13 +25,20 @@ def __init__(self, tenant):
...
 
-        client = bigquery.Client()
+        columns = ["name", "FORMAT_DATETIME('%Y-%m-%d %H:%M:%S', timestamp) AS datetime",
```
Comment (NoahNsimbe) [inline src/analytics/api/models/events.py:34]: Formatting the datetime

File: src/analytics/api/models/events.py [hunk 3]
```diff
@@ -42,9 +49,10 @@ def from_bigquery(cls, tenant, sites, start_date, end_date, frequency, pollutant
         job_config = bigquery.QueryJobConfig()
         job_config.use_query_cache = True
 
-        query = client.query(QUERY, job_config)
+        dataframe = bigquery.Client().query(QUERY, job_config).result().to_dataframe()
+        dataframe.sort_values(["name", "datetime"], ascending=True, inplace=True)
```
Comment (NoahNsimbe) [inline src/analytics/api/models/events.py:53]: Sorting the values by site name and datetime

**Review by Baalmart — APPROVED** [review]
LGTM

**Review by sserurich — APPROVED** [review]
LGTM

**Pull request merged**